doc/source/index.rst

```diff
@@ -112,6 +112,153 @@ Ray provides Python and Java API. And Ray uses Tasks (functions) and Actors (Cla
           }
         }
 
+  .. group-tab:: C++ (EXPERIMENTAL)
+
+    The C++ Ray API is currently experimental with limited support. You can track its development `here <https://github.com/ray-project/ray/milestone/17>`__ and report issues on GitHub.
+    Run the following commands to get started:
+    - Build ray from source with *bazel* as shown `here <https://docs.ray.io/en/master/development.html#building-ray-full>`__.
+    - Run `cd ray/cpp`.
+    - Run `cp dev_BUILD.bazel BUILD.bazel`.
+    - Modify `src/ray/example.cc`.
+    - Run `ray stop`.
+    - Run `bazel build //cpp:example`.
+    - Run `bazel run //cpp:example`.
+
+    .. code-block:: C++
```

Comment:
the rest of the code base use literal-include so you don't have to copy changes around.https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-literalinclude

Reply:
I am not sure I understand, I just followed the Java and Python code-block. Can you elaborate on what exactly you think we should do here? @simon-mo

Reply:
I'm trying to say that the content of this code block is a duplicate of `cpp/src/ray/example.cc` so we should just include the file using literalinclude. _The code block is used for snippets, not entire file_. Using literal include makes the documentation more maintainable because if you want to edit `cpp/src/ray/example.cc`, you don't need to edit two places.

Reply:
Oh, gotcha!